app/routes/v1/channelsStream.js

```diff
@@ -14,7 +14,8 @@ const crypto = require('crypto');
 const algorithm = 'aes-256-cbc';
 
 const getOrg = require('../../utils/orgs.js').getOrg;
-const requireAuth = require('../../utils/api_utils.js').requireAuth;
+const { ACTIONS, TYPES } = require('../../utils/auth.consts');
+const { rbacAuth } = require('../../utils/auth');
```

Comment:
We should make this pluggable (either here or in the `../../utils/auth`) module.  Then an environment variable should be used to override what we put in the `require` statement.

Reply:
implemented.